pkg/operators/ebpf/struct.go

```diff
@@ -200,25 +213,31 @@ func (i *ebpfInstance) getFieldsFromMember(member btf.Member, fields *[]*Field,
 		Ellipsis:  metadatav1.EllipsisEnd,
 	}
 
-	newField := func(size uint32, reflectType reflect.Type) *Field {
-		return &Field{
+	newField := func(size uint32, kind api.Kind) *Field {
+		field := &Field{
 			Field: metadatav1.Field{
 				Name:       prefix + member.Name,
 				Attributes: defaultAttributes,
 			},
 			Size:   size,
 			Tags:   tags,
-			Type:   reflectType,
 			Offset: offset + member.Offset.Bytes(),
 			parent: parent,
 			name:   member.Name,
+			kind:   kind,
 		}
+
+		field.Field.Attributes.Width = uint(columns.GetWidthFromType(refType.Kind()))
+
+		return field
 	}
 
 	// Flatten embedded structs
 	if t, ok := member.Type.(*btf.Struct); ok {
 		// Add outer struct as well
-		field := newField(t.Size, reflect.ArrayOf(int(t.Size), reflect.TypeOf(uint8(0))))
+		// TODO: what to put here?
```

Comment:
How about adding a new `Kind_Bytes` and use `Kind_Invalid` when it's just that (virtual, containers, etc., basically anything without underlying memory).

Reply:
Makes sense. I added it.